src/cnv/pyproject.toml

```diff
@@ -3,12 +3,9 @@ name = "ugbio_cnv"
 version = "1.17.3-0"
 requires-python = ">=3.11"
 dependencies = [
-    "ugbio_core[vcfbed]",
-    "seaborn>=0.13.2",
+    "ugbio_core[vcfbed,viz,parquet]",
```

Comment:
Definitely too much division. 

What does "parquet" even mean? What kind of dependency is this?